test/junit/scala/collection/StringOpsTest.scala

```diff
@@ -93,4 +93,8 @@ class StringOpsTest {
     assertEquals("", "a" * 0)
     assertEquals("", "a" * -1)
   }
+
+  @Test def withFilterAndThenMap(): Unit = {
+    assertEquals("hello".withFilter(_ != 'e').map(_.toUpper), "HLLO")
```

Comment:
The junit convention is `assertEquals(expected, actual)`. For `Future(reference)`.

Reply:
oops.. Should I restore the branch, swap arguments and then add it to the same commit? I believe it will badly work for merged branch. Or should I correct it in another commit/PR?

Reply:
No big deal, it only matters for the message if it fails. I'm hyperalert because my office uses testng, with the arguments in the other order ("the wrong order"). I only mention it as a PSA.